styles.css

```diff
@@ -442,14 +430,6 @@ Alerts  */
   padding-left: 1rem;
 }
 
-/* .dismiss-card:hover {
-  box-shadow: 0 0 0 4px hsla(0, 0%, 80%), 0 0 0 6px hsla(0, 0%, 75%);
-  /* transform: translateZ(1);
-  opacity: 0.5; */
-/* z-index: 2; */
-/* box-shadow: 5px 10px #888888;
-} */
-
 .card-image {
   width: 100%;
```

Comment:
I think `max-width:100%` kiya toh `height:auto` is not necessary. Try it out!

Reply:
Ya, I guess that is surplus. Removed it too.